rfc/0007-community-team.md

```diff
@@ -233,7 +233,7 @@ The election has **failed** when one of the following is true:
 * A week has passed and less than 75% of the community team and administrative board has voted
 * A week has passed, at least 75% of the community team and administrative board has voted, and those that have
   voted have reached a consensus in opposition of the candidate - or a consensus cannot be reached
-* The candidate receives more negative votes than positive votes
+* An absolute majority of the community team and administrative board has voted against the candidate
```

Comment:
I would change it to something like this:
```suggestion
* An absolute majority of the community team and administrative board who voted, has voted against the candidate, after 3 days of the vote being open.
```

Reply:
Your change goes completely against the purpose of this PR, as that is a simple majority, and is actually covered by this portion of the existing RFC:

> A week has passed, at least 75% of the community team and administrative board has voted, and those that have
  voted have reached a consensus in opposition of the candidate

Essentially, this PR breaks down to:
- If 75% of the people who *actually voted* vote negatively, the vote fails (existing rule)
- *Additionally*, if 51% of everyone who *could* vote votes negatively, the vote fails

Reply:
I am worried about a situation where if the project is already dying, this system will cause it to enter a spiral which kills it faster. If we ever reach that 25% threshold of community team and admin team members becoming inactive for some reason, it would be impossible for any election to pass and to elect any active team members.

Reply:
If anything, this change makes that *less* likely to be a problem, as the inactive members cannot vote explicitly against the candidate. All this change does is increase the threshold for a failed vote from "If there are ever more negative voters than positive voters, it fails" to "If more than half of everyone who *could* vote *actively votes against the candidate*, the vote fails".

Reply:
Wouldn't this whole issue be covered by the fact that people deemed too inactive to be beneficial to the project can be democratically voted out of their position on the team, and/or someone more active can be added to the team?

Reply:
Hmm, I think for clarity the "a vote passes when" section needs to be amended to reflect this. Because right now there is no bullet point under that section that says a vote could pass if less than 75% of the team is active.

Reply:
In my opinion, it would be better to have an RFC which centralizes and consolidates voting processes. Such an RFC would tie in to RFC 7 and to [RFC 6: Governmental Structure](https://github.com/QuiltMC/rfcs/pull/6) (such as for votes for member removal or team creation).

This PR is currently scoped to amending the specific "negative votes greater than positive votes" clause, which is a more pressing matter if an election were to be held (because this can be triggered by one team member, vs the 25% membership count that would block a vote if they are inactive). 

While I am not against clarifying how a vote can pass if less than 75% of the team has voted and 25%+ are 'inactive', as I've said, it would best be resolved by an RFC which details the voting process (such as inactivity, abstaining, majorities required for different actions, uncovered situations, etc) so everything voting-related is linked in one central place.